hadoop-ozone/ozone-manager/src/test/java/org/apache/hadoop/ozone/om/request/snapshot/TestOMSnapshotPurgeRequestAndResponse.java

```diff
@@ -260,6 +262,8 @@ public void testValidateAndUpdateCache() throws Exception {
     for (Path checkpoint : checkpointPaths) {
       assertFalse(Files.exists(checkpoint));
     }
+    assertEquals(initialSnapshotPurgeCount + 1, omMetrics.getNumSnapshotPurges());
+    assertEquals(initialSnapshotPurgeFailCount, omMetrics.getNumSnapshotPurgeFails());
```

Comment:
If it isn't too much hassle we could inject failure to test the failure counter as well. :) Same for `TestOMSnapshotSetPropertyRequestAndResponse`

Reply:
Added tests as suggested.